include/mbedtls/gcm.h

```diff
@@ -298,9 +281,9 @@ int mbedtls_gcm_update_ad(mbedtls_gcm_context *ctx,
  *                      is greater than zero, this must be a writable buffer of
  *                      of at least \p output_size bytes.
  * \param output_size   The size of the output buffer in bytes.
- *                      See the function description regarding the output size.
  * \param output_length On success, \p *output_length contains the actual
- *                      length of the output written in \p output.
+ *                      length of the output written in \p output, which equals
```

Comment:
This should say: "On success, \p *output_length contains the actual length of the output written in \p output, which is equal to \p input_length."

Reply:
Thanks. It's fixed.